.test-infra/jenkins/job_LoadTests_GBK_Java_spark_structured_streaming.groovy

```diff
@@ -0,0 +1,208 @@
+                            publishToBigQuery     : false,
+                            bigQueryDataset       : datasetName,
+                            bigQueryTable         : "java_SparkStructuredStreaming_${mode}_GBK_6",
+                            sourceOptions         : """
+                                            {
+                                              "numRecords": 20000000,
+                                              "keySizeBytes": 10,
+                                              "valueSizeBytes": 90,
+                                              "numHotKeys": 200,
+                                              "hotKeyFraction": 1
+                                            }
+                                       """.trim().replaceAll("\\s", ""),
+                            fanout                : 1,
+                            iterations            : 4,
+                            streaming             : isStreaming
+                    ]
+            ],
+            [
+                    title          : 'Load test: reiterate 4 times 2MB values',
+                    test           : 'org.apache.beam.sdk.loadtests.GroupByKeyLoadTest',
+                    runner         : CommonTestProperties.Runner.SPARK_STRUCTURED_STREAMING,
+                    pipelineOptions: [
+                            project               : 'apache-beam-testing',
+                            appName               : "load_tests_Java_SparkStructuredStreaming_${mode}_GBK_7",
+                            tempLocation          : 'gs://temp-storage-for-perf-tests/loadtests',
+                            publishToBigQuery     : false,
+                            bigQueryDataset       : datasetName,
+                            bigQueryTable         : "java_SparkStructuredStreaming_${mode}_GBK_7",
+                            sourceOptions         : """
+                                            {
+                                              "numRecords": 20000000,
+                                              "keySizeBytes": 10,
+                                              "valueSizeBytes": 90,
+                                              "numHotKeys": 10,
+                                              "hotKeyFraction": 1
+                                            }
+                                       """.trim().replaceAll("\\s", ""),
+                            fanout                : 1,
+                            iterations            : 4,
+                            streaming             : isStreaming
+                    ]
+            ]
+    ]
+}
+
+def batchLoadTestJob = { scope, triggeringContext ->
+    def datasetName = loadTestsBuilder.getBigQueryDataset('load_test', triggeringContext)
+    loadTestsBuilder.loadTests(scope, CommonTestProperties.SDK.JAVA, loadTestConfigurations('batch', false, datasetName), "GBK", "batch")
+}
+
+PhraseTriggeringPostCommitBuilder.postCommitJob(
```

Comment:
This runs only when we type `Run load test...` in the comment. Should we add the Cron-trigged version?

Reply:
same as above ;)

Reply:
cron added